modules/meb_api/lib/authentication_token_service.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module MebApi
+  class AuthenticationTokenService
+    ALGORITHM_TYPE = 'RS256'
```

Comment:
Do you want to namespace the services, config, etc classes with the v0 namespace?